# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu_bindings/Cargo.toml
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 gfx/wgpu_bindings/Cargo.toml
--- a/gfx/wgpu_bindings/Cargo.toml	Fri Oct 03 18:01:30 2025 +0000
+++ b/gfx/wgpu_bindings/Cargo.toml	Fri Oct 03 18:08:29 2025 +0000
@@ -17,7 +17,7 @@
 [dependencies.wgc]
 package = "wgpu-core"
 git = "https://github.com/gfx-rs/wgpu"
-rev = "ffd4e9fb53a98f8ad67bab8c5d2b42cc9dbdff24"
+rev = "8c4aebc0c1b75e3f7412dfc180745a0717fc30ec"
 features = ["serde", "trace", "strict_asserts", "wgsl", "api_log_info"]
 
 # We want the wgpu-core Metal backend on macOS and iOS.
@@ -25,32 +25,32 @@
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.wgc]
 package = "wgpu-core"
 git = "https://github.com/gfx-rs/wgpu"
-rev = "ffd4e9fb53a98f8ad67bab8c5d2b42cc9dbdff24"
+rev = "8c4aebc0c1b75e3f7412dfc180745a0717fc30ec"
 features = ["metal"]
 
 # We want the wgpu-core Direct3D backends on Windows.
 [target.'cfg(windows)'.dependencies.wgc]
 package = "wgpu-core"
 git = "https://github.com/gfx-rs/wgpu"
-rev = "ffd4e9fb53a98f8ad67bab8c5d2b42cc9dbdff24"
+rev = "8c4aebc0c1b75e3f7412dfc180745a0717fc30ec"
 features = ["dx12"]
 
 # We want the wgpu-core Vulkan backend on Linux and Windows.
 [target.'cfg(any(windows, all(unix, not(any(target_os = "macos", target_os = "ios")))))'.dependencies.wgc]
 package = "wgpu-core"
 git = "https://github.com/gfx-rs/wgpu"
-rev = "ffd4e9fb53a98f8ad67bab8c5d2b42cc9dbdff24"
+rev = "8c4aebc0c1b75e3f7412dfc180745a0717fc30ec"
 features = ["vulkan"]
 
 [dependencies.wgt]
 package = "wgpu-types"
 git = "https://github.com/gfx-rs/wgpu"
-rev = "ffd4e9fb53a98f8ad67bab8c5d2b42cc9dbdff24"
+rev = "8c4aebc0c1b75e3f7412dfc180745a0717fc30ec"
 
 [dependencies.wgh]
 package = "wgpu-hal"
 git = "https://github.com/gfx-rs/wgpu"
-rev = "ffd4e9fb53a98f8ad67bab8c5d2b42cc9dbdff24"
+rev = "8c4aebc0c1b75e3f7412dfc180745a0717fc30ec"
 features = ["device_lost_panic", "internal_error_panic"]
 
 [target.'cfg(windows)'.dependencies]